src/main/java/org/gridsuite/shortcircuit/server/ShortCircuitController.java

```diff
@@ -59,11 +59,12 @@ public ResponseEntity<UUID> runAndSave(@Parameter(description = "Network UUID")
                                            @Parameter(description = "Result receiver") @RequestParam(name = "receiver", required = false) String receiver,
                                            @Parameter(description = "reportUuid") @RequestParam(name = "reportUuid", required = false) UUID reportUuid,
                                            @Parameter(description = "reporterId") @RequestParam(name = "reporterId", required = false) String reporterId,
+                                           @Parameter(description = "The type name for the report") @RequestParam(name = "reportType", required = false) String reportType,
```

Comment:
Add defaultValue

Reply:
I cant put a default value here: I dont if it's AllUses or OneBus case.
So I made it in the workerService